### ISSUE TITLE
T-Mobile T-Vision

### ISSUE THREAD

**Issue by vlrs** [StevenBlack/hosts#1462]
Hello. When trying to view the TV channels offered by T-Mobile T-Vision, the page won't load. You put in a zip code, but clicking the See Channels won't load anything.
https://www.t-mobile.com/tvision/channel-lineup

Whitelisting tags.tiqcdn.com fixes it.

I searched tags.tiqcdn.com and it seems it was an MVPS link before, but MVPS disabled it because it affects Intel.com.

Thank you.

**Comment by dnmTX** [comment]
@vlrs there is one more additional entry of `tags.tiqcdn.com` in yoyo.org's list.
You can report it to the curator directly: `pgl@yoyo.org` or on [twitter](https://twitter.com/pgl)

**Comment by vlrs** [comment]
> @vlrs there is one more additional entry of `tags.tiqcdn.com` in yoyo.org's list.
> You can report it to the curator directly: `pgl@yoyo.org` or on [twitter](https://twitter.com/pgl)

Thank you. Will do.

**Comment by dnmTX** [comment]
Peter @pgl,could you please advise on this one as it's been sitting here for a while now. Thank you 👍

**Comment by pgl** [comment]
Hi there @dnmTX. I wasn't aware of this ticket, I don't monitor issues from this repo specifically and wasn't pinged until you just did.

I've replied to emails about this domain, and it's overwhelmingly being used as a tracker. I actually tried whitelisting it and browsing for a while, and the number of tracking domains contacted was amazing.

Take a look at tealium.com - it's all about tracking users. Their big product is called a "Customer Data Hub". This isn't _for_ customers, it's data _on_ customers. I'll stop here but there's plenty of examples of where their focus is.

If a site is broken because `tags.tiqcdn.com` is broken, I encourage people to contact the owners of that site and ask them to design it so it doesn't fail when remote scripts aren't loaded.
